public/sass/pages/_dashboard.scss

```diff
@@ -57,6 +57,7 @@ div.flot-text {
   min-height: 9px;
   padding-top: 4px;
   cursor: pointer;
+  word-wrap: break-word;
```

Comment:
this does not look right. The dashboard title should never wrap (the header cannot be higher or things break).